### ISSUE TITLE
Apache 500 error with debug = false

### ISSUE THREAD

**Issue by Emil-G** [tobami/codespeed#258]
Hi,

When copying the Sample project and doing two changes to the `settings.py`:
changing debug: 
`DEBUG = False`
allowing hosts:
`ALLOWED_HOSTS = ['*']`

and starting that with Apache if fails with a 500 internal server error. Works when debug = true.  Looks like there could be a lot of different reasons as to why this is happening. 
Any clue into why?

**Comment by tobami** [comment]
Is this fixed by #257?

**Comment by Emil-G** [comment]
@tobami : No, #257 is needed to get the application to load on Apache, it wont work without changing that one.
From what I could understand, ` DEBUG = True` ignores any strict checks/validation of the code, so when setting `False`,  it fails. However, I don't have any idea as to which part of code that it failing the checks. It was just a clean git pull from master, so no customization was done here (except for the Django loading change introduced in 1.7 I belive.)
I did this setup as a favor for a dev in my company, he did not care with it running in Debug mode. So you can just close this unless you want to debug where it fails the Django validation.
